scripts/coverage/run_all_coverage.sh

```diff
@@ -21,12 +21,18 @@
 set -e
 set +o posix
 
+#Check if codacy token as an argument is passed
+if [[ $# -eq 0 ]] ; then
+    echo 'Error: No arguments supplied,please pass codacy token as an argument to the script'
+    exit 1
+fi
+
 # variables
 SCRIPT_PATH=$(readlink -f "$0")
 SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
 SRC_DIR="$(dirname "$(dirname "$SCRIPT_DIR" )")"
 # Default codacy project token for cortx-s3server
-CODACY_PROJECT_TOKEN=7a64ef82e1a049ab898fe838f1320594
+CODACY_PROJECT_TOKEN=$1
```

Comment:
There is already a way "-t" in getopt to accept `CODACY_PROJECT_TOKEN`. Couldn't we use that instead (make it mandatory option) and here set this variable="" ?
I can help you with this.

else if we are using $1, then `shift 1;` is necessary ...for getopt to fetch further arguments